### PR TITLE
Extract validator from AdminBundle

### PR DESCRIPTION
**Pull request by Shine-neko**
See https://github.com/sonata-project/SonataMediaBundle/issues/680

### CODE REVIEW

File: Validator/Constraints/InlineConstraint.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    protected $service;
+
+    protected $method;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function validatedBy()
+    {
+        return 'sonata.admin.validator.inline';
```
Comment (rande) [inline Validator/Constraints/InlineConstraint.php]: This should be renamed

**Comment by rande** [comment]
You also need to migrate unit test ;)

**Comment by Shine-neko** [comment]
This is in progress

**Comment by Shine-neko** [comment]
Thanks

**Comment by rande** [comment]
@Shine-neko you forgot the xml service definition, see : https://github.com/sonata-project/SonataDoctrineORMAdminBundle/issues/421

**Pull request merged**